cmd/manager/main.go

```diff
@@ -122,6 +120,11 @@ func init() {
 		certificates.DefaultRotateBefore,
 		"Duration representing how long before expiration TLS certificates should be reissued",
 	)
+	Cmd.Flags().Bool(
+		EnableWebhooksFlag,
```

Comment:
This somewhat duplicates what we used to express with the `webhook` role. I am fine with moving away from the operator roles concept but I think we should not have both the role and the explicit flag at the same time.

Reply:
Ah that is an oversight on my part, I forgot about the roles. I'll change this, we'll just want to change the default to not-webhook now I think. And change it back when we have automatic webhook configuration